packages/site/.env.local

```diff
@@ -3,4 +3,7 @@ MOSAIC_SNAPSHOT_DIR=snapshots/latest
 OPTIMIZE_IMAGES=false
 NEXTAUTH_URL=http://localhost:3000
 NODE_ENV=development
-MOSAIC_ENABLE_SOURCE_PUSH=true
\ No newline at end of file
+MOSAIC_ENABLE_SOURCE_PUSH=true
+NEXTAUTH_URL=http://localhost:3000/
```

Comment:
repeat of above? or you need one with a slash?

Reply:
no copy & paste error...will remove in next PR

Reply:
think it works with either